### DISCUSSION TITLE
Patch method bad behavior with virtual fields

### DISCUSSION THREAD

**Discussion by carlos9108** [binaryk/laravel-restify#546] [Q&A] [answered]
I have this field definition:
```
Field::new('tag_type_id', fn() => $this->model()->tagTypes->pluck('id'))
                ->action(new AttachModelAction('tag_type_id', 'tagTypes')),
```
whose AttachModelAction basically does this
```
 $value = $request->input('tag_type_id') ?? [];
  $asset->$this->tagTypes()->sync($value);
```

Now, this works great with POST/PUT method, but PATCH completely breaks this way:

` column \"tag_type_id\" of relation \"subfamily_types\" does not exist\nLINE 1: update \"subfamily_types\" set \"tag_type_id\" = $1, \"updated_at...\n                                     ^ (SQL: update \"subfamily_types\" set \"tag_type_id\" = [3,1], \"updated_at\" = 2023-03-13 04:38:51 where \"id\" = 177)",`

I mean, obviously tag_type_id is a virtual field, yet patch method seems to ignore this. Any idea why????

**Comment by dsindrilaru** [comment, marked as answer]
Hi @carlos9108! Thanks for sharing this issue, a fix for it has been implemented in the 7.11.0 release.
